libnopegl/src/node_texture.c

```diff
@@ -571,6 +630,20 @@ const struct node_class ngli_texture2d_class = {
     .file      = __FILE__,
 };
 
+const struct node_class ngli_texture2d_array_class = {
```

Comment:
can you name this `ngli_texture2darray_class` for consistency with all the other classes?

Reply:
Done.